### ISSUE TITLE
`snip` throws Lisp exception: `Undefined function BOXER::TRANSLATE-GRAPHICS-COMMAND`

### ISSUE THREAD

**Issue by rigdern** [boxer-project/boxer-sunrise#108]
# Repro steps
I reproed this with Boxer `3.4.19 2024-06-24`

1. Create a turtle by pressing `option-t`
2. This created a graphics box. Name it `graphics`
3. Draw a rectangle with `stamp-rectangle 50 50`
4. Try to snip the graphics with `snip graphics 0 0 10 10`

**Unexpected result:** `snip` results in a Common Lisp exception:

```
Lisp Error:Undefined function BOXER::TRANSLATE-GRAPHICS-COMMAND
  called with arguments (#(42 0.0 0.0 50 50) -145 -95). 
```

Alternatively you can repro it with [snip repro.box.zip](https://github.com/user-attachments/files/16385332/snip.repro.box.zip). Screenshot:

<img width="775" alt="image" src="https://github.com/user-attachments/assets/d4006278-81e0-47a0-a664-de7f809cf089">

**Comment by sgithens** [comment]
I believe this has actually been fixed in release 3.4.20 by way of another issue.

**Comment by rigdern** [comment]
I confirmed this no longer repros in Boxer version `3.4.20 2024-08-06`
